boogie-routes/src/test/java/org/mitre/tdp/boogie/alg/chooser/GraphicalRouteChooserTest.java

```diff
@@ -108,7 +108,7 @@ private static RouteTokenResolver apfResolver() {
     Leg l3 = TF("GRRDR", 0.0, 2.0);
     Leg l4 = TF("VNY", 0.0, 3.0);
 
-    Transition t = transition("BLSTR1", TransitionType.COMMON, ProcedureType.SID, Arrays.asList(l1, l2, l3, l4));
+    Transition t = transition("BLSTR1", "KIND", TransitionType.COMMON, ProcedureType.SID, Arrays.asList(l1, l2, l3, l4));
```

Comment:
by default this airport name was `"FOO"`, which failed the filter added in this PR